src/lib.rs

```diff
@@ -462,15 +464,16 @@ fn graph_is_isomorphic(
 /// :returns: ``True`` if there is a subgraph of `first` isomorphic to `second`,
 ///     ``False`` if there is not.
 /// :rtype: bool
```

Comment:
Do you want to update the docstrings here (and for the graph function) to add the new parameter and the description of node-induced too?